examples/models/README.md

```diff
@@ -0,0 +1,76 @@
+# Nvidia backend implementation notes
+
+The Nvidia backend for DNNL can be exposed to the user via the `dnnl::engine::kind::gpu` engine kind. Currently, for the case when user's system has both Intel and Nvidia GPUs, 
+`DNNL_GPU_VENDOR=NVIDIA` Flag is used in CMake, since the devices are clustered based on the device vendor ID and index pattern can not be used to distinguish between Intel GPU and Nvidia GPU.
+
+## Requirements for Building from source
+
+- oneDNN Nvidia backend requires Intel oneAPI DPC++ Compiler, details of aquiring and installing it is available https://github.com/mehdi-goli/oneDNN/blob/codeplay-onednn-nvidia-support/README.md
+- git-lfs is required to checkout the build
+
+```
+$ git clone https://github.com/mehdi-goli/oneDNN.git -b codeplay-onednn-nvidia-support
```

Comment:
Still references the other repo

Reply:
Also have they not already cloned the repo?

Reply:
Fixed the repo. 
This is the section which talks about which branch in the repo to use. The previous section was only dependencies.